test/runtests.jl

```diff
@@ -837,60 +838,48 @@ end
     @testset "Examples" begin
         @testset "Referencing symbols from other packages" begin
             @testset "Referencing symbols with import" begin
-                Pkg.add("TimeZones")
-
                 pnums = addprocs(3)
                 @everywhere using Dispatcher
-                @everywhere import TimeZones
+                @everywhere import Compat
 
                 try
                     ctx = @dispatch_context begin
-                        a = @op TimeZones.ZonedDateTime(
-                            DateTime(2014,1,1), TimeZones.TimeZone("Europe/Warsaw")
-                        )
-                        b = @op TimeZones.ZonedDateTime(
-                            DateTime(2014,1,2), TimeZones.TimeZone("Europe/Warsaw")
-                        )
-                        b = @op TimeZones.astimezone(a, TimeZones.TimeZone("Asia/Tokyo"))
-                        d = Op(max, a, b)
+                        a = @op Compat.Float64(2.0)
+                        b = @op Compat.Nullable(2.0)
+                        c = @op Compat.isequal(a, b)
+                        d = @op Compat.isnull(b)
                     end
 
                     exec = ParallelExecutor()
                     (results,) = run!(exec, ctx)
                     @test !iserror(results)
                     run_future = unwrap(results)
                     @test isready(run_future)
-                    @test fetch(run_future) == TimeZones.ZonedDateTime(
-                        DateTime(2014,1,2), TimeZones.TimeZone("Europe/Warsaw")
-                    )
+                    @test fetch(run_future) == false
                 finally
                     rmprocs(pnums)
                 end
             end
 
             @testset "Referencing symbols with using" begin
-                Pkg.add("TimeZones")
-
                 pnums = addprocs(3)
                 @everywhere using Dispatcher
-                @everywhere using TimeZones
+                @everywhere using Compat
 
                 try
                     ctx = @dispatch_context begin
-                        a = @op ZonedDateTime(DateTime(2014,1,1), TimeZone("Europe/Warsaw"))
-                        b = @op ZonedDateTime(DateTime(2014,1,2), TimeZone("Europe/Warsaw"))
-                        b = @op astimezone(a, TimeZone("Asia/Tokyo"))
-                        d = Op(max, a, b)
+                        a = @op @compat Float64(2.0)
```

Comment:
This test can't be a macro, as those will be evaluated on the host process. I think I led you down a bad path by suggesting Compat as the test module :( sorry. Use Iterators.jl instead.